zproject/settings.py

```diff
@@ -1302,6 +1302,6 @@ def get_secret(key):
 # This is a debugging option only
 PROFILE_ALL_REQUESTS = False
 
-CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com'))
+CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com', 'welcome-bot@zulip.com'))
```

Comment:
I think leaving it realm-specific is probably good. Cross realm is used to send messages from one realm to a different realm, which is not needed here.